### ISSUE TITLE
Support for `jbake -s`

### ISSUE THREAD

**Issue by The28AWG** [jbake-org/jbake-gradle-plugin#32]
Hello!
Pls, create task "preview"

**Comment by antonmry** [comment]
As workarround: you may use livereload plugin. If you want to have it updated with changes in real-time:

```sh
./gradlew -t bake
```

To run it locally:

```sh
./gradlew liveReload 
```
and open http://localhost:35729 with your browser. It should refresh with any change.

An example here: https://github.com/vigojug/vigojug.github.io/blob/source/build.gradle

**Comment by The28AWG** [comment]
Thanks!

**Comment by msgilligan** [comment]
See PR #5 (which is too old to be mergeable)

**Comment by ancho** [comment]
You can use the gretty plugin to serve your baked site out of gradle as a workaround.
See https://github.com/ancho/calmdev-website/blob/master/build.gradle#L32 for an example. You can ignore the config stuff there. Just set `httpPort` and `contextPath`, if you have one, and run `gradle jettyRun`.

Just a scetch:

```
plugins {
    id 'org.akhikhl.gretty' version '1.4.0'
}

apply plugin: 'war'

gretty {
   httpPort = 8820
   contextPath = '/blog'
   extraResourceBases = [bake.output]
}
```
See http://akhikhl.github.io/gretty-doc/ for more information. Hope that helps.

Maybe we should integrate that into the plugin instead of using the jbake built in?

**Comment by aalmiray** [comment]
Solved when #2 was merged
